runner/tests/tests.rs

```diff
@@ -13,6 +13,30 @@ fn test_choi_jamiolkowski_isomorphism() -> Result<(), String> {
     run_bitcode(bitcode, None)
 }
 
+#[test]
+fn test_full_qir_simple() -> Result<(), String> {
+    let bitcode = include_bytes!("resources/full-qir.bc");
+    run_bitcode(bitcode, Some("QIR_App_Test__Simple"))
+}
+
+#[test]
+fn test_full_qir_other() -> Result<(), String> {
+    let bitcode = include_bytes!("resources/full-qir.bc");
+    run_bitcode(bitcode, Some("QIR_App_Test__Other"))
+}
+
+#[test]
+fn test_qir_testing() -> Result<(), String> {
+    let bitcode = include_bytes!("resources/qir-testing.bc");
```

Comment:
Do we have any description of that this file (e.g. `qir-testing.bc`) contains or what this is checking for specifically? (i.e. does this file contain all valid opcodes or something?)

Reply:
I've refactored some of the tests and added descriptions for each to help clarify!